### DISCUSSION TITLE
Move to modbusproxy - How to change stored IP?

### DISCUSSION THREAD

**Discussion by gimmel** [wlcrs/huawei_solar#645] [Setting up the integration]
I’m using the Huawei Solar integration and it works well so far. Now I wanted to add the evcc add-on, which also uses the modbus. But Huawei only allows one modbus connection, so I have to add the modbus-proxy add-on. Now I have to change the IP of the modbus from inverter to proxy, but I don't know where. I did not found any settings to edit the login data. And I don’t want to remove and re-add the plugin to keep the data recorded so far.

Where can I change the ip address of a device in the Huawei Solar integration?
I would to like to get to this screen even after all was set up.
![](https://raw.githubusercontent.com/wlcrs/huawei_solar/1.3.3/images/network-configuration.png)

**Comment by malgates** [comment]
Th only option is to delete your Inverter and reinstall it again. Did the same. Not able to bring it up: https://github.com/wlcrs/huawei_solar/discussions/266
